db/migrate/20231211113423_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateGreetings < ActiveRecord::Migration[7.1]
+  def change
+    create_table :greetings do |t|
+      t.string :message
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] I encountered an error in naming convention for running your migrations. 
Kindly fix this 

### View:

![image](https://github.com/mumbereh/Hello_Webpack/assets/50990925/04641227-fa9c-4dab-aa29-ddc6b39a312c)